beebop/assignClusters.py

```diff
@@ -7,12 +7,13 @@
 import re
 import os
 import pickle
-from typing import Union
+from typing import Union, ItemsView
```

Comment:
There's a suggestion here that this is a deprecated alias of `collections.abc.ItemsView`: https://docs.python.org/3/library/typing.html